src/main/java/seedu/trackbeau/model/ModelManager.java

```diff
@@ -93,11 +96,22 @@ public boolean hasCustomer(Customer customer) {
         return trackBeau.hasCustomer(customer);
     }
 
+    @Override
+    public void deleteBooking(Booking target) {
+        trackBeau.removeBooking(target);
+    }
+
     @Override
     public void deleteCustomer(Customer target) {
         trackBeau.removeCustomer(target);
     }
 
+    @Override
+    public void addBooking(Booking booking) {
+        trackBeau.addBooking(booking);
+        updateFilteredCustomerList(PREDICATE_SHOW_ALL_CUSTOMERS);
```

Comment:
Is this supposed to be updateFilteredBookingList(PREDICATE_SHOW_ALL_BOOKINGS);?

Reply:
Updated